### PR TITLE
Update sdf.md

### PR DESCRIPTION
**Pull request by mjkoster**
fix the name resolution section

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -25,14 +25,20 @@ The JSON format of an SDF definition is described in this document.
...
+          "enum": [
+            { "on":1 },
+            { "off":0 }
+          ]
         }
       },
       "odmAction": {
-        "on": {},
-        "off": {}
+        "on": {"id":3},
```
Comment (gerickson) [inline sdf.md]: Is this the revised format of enumerations?

File: sdf.md [hunk 2]
```diff
@@ -102,29 +108,39 @@ For example, an Object definition looks like this:
...
+```
+Would be expanded into:
+```
+"https://example.com/#temperatureData"
+```
 
 ### Target namespace
 
-The target namespace is the namespace into which the defined terms are added. The target namespace is defined by the default namespace, or by an explicit prefix on the identifier separated by a semicolon ":".
+The target namespace is the namespace into which the defined terms are added. The target namespace is defined by the default namespace, or by an explicit prefix on the identifier using semicolon ":".
```
Comment (gerickson) [inline sdf.md]: Colon? The text says 'semicolon ":'' but shows a colon.

File: sdf.md (hunk 1 above)
Comment (mjkoster) [reply sdf.md]: This is for "id" being a required quality of all definitions. There is another change in the examples for enum format.

File: sdf.md (hunk 2 above)
Comment (mjkoster) [reply sdf.md]: fixed

**Comment by mjkoster** [comment]
OK To Merge?

**Comment by mjkoster** [comment]
@gerickson Does this look OK wrt your questions?

File: sdf.md (hunk 2 above)
Comment (gerickson) [reply sdf.md]: Thanks.

**Comment by gerickson** [comment]
> @gerickson Does this look OK wrt your questions?

LGTM.

**Pull request merged**